store/src/main/scala/net/spals/leagur/store/hyperdex/impl/HyperdexStore.scala

```diff
@@ -16,26 +20,54 @@ import scala.collection.JavaConverters._
 @AutoBindSingleton(classOf[Store])
 class HyperdexStore extends Store {
 
+  @NotNull
+  @Configuration(value = "hyperdex.store.port")
+  private var hyperdexPort: Int = 0
```

Comment:
Example of configuration injection. These values will be automatically injected into the class after Governator creates it.